examples/demo-react-native-jest/e2e/init.js

```diff
@@ -1,14 +1,18 @@
-
-const detox = require('detox');
 const config = require('../package.json').detox;
 const adapter = require('detox/runners/jest/adapter');
+const traceAdapter = require('detox/runners/jest/traceAdapter');
 
 // Set the default timeout
 jest.setTimeout(300000);
 jasmine.getEnv().addReporter(adapter);
 
+// This takes care of generating status logs on a per-test basis.
```

Comment:
If we change anything in the init code, we should not forget to update the code here too.